pyproject.toml

```diff
@@ -160,6 +160,28 @@ filterwarnings = [
     "ignore:SelectableGroups dict interface is deprecated:DeprecationWarning"
 ]
 
+[tool.setuptools]
+zip-safe = false
+include-package-data = false
+
+[tool.setuptools.packages]
+find = {namespaces = false}
+
+[tool.setuptools.package-data]
+qcodes = [
+    "monitor/dist/*",
+    "monitor/dist/js/*",
+    "monitor/dist/css/*",
+    "configuration/*.json",
+    "instrument/sims/*.yaml",
+    "tests/dataset/fixtures/2018-01-17/*/*",
+    "tests/delegate/data/*.yml",
+    "tests/drivers/auxiliary_files/*",
+    "py.typed",
+    "dist/schemas/*",
+    "dist/tests/station/*",
+]
```

Comment:
If we move this to MANIFEST.in we can set include-package-data to True (The default) and remove that line too